### PR TITLE
Add index GET route for liveness/readiness probes

### PR DESCRIPTION
**Pull request by AparnaKarve**
- Added a new index GET route to be used for liveness/readiness of the pod
 This route would ping the next service for availability, which is `aiops-publisher` 

- Added tests for different scenarios of the index GET route

(Also included some pylama linter fixes that were not applied to `volume_type_validation` the first time around)

Requires: https://github.com/ManageIQ/aiops-publisher/pull/22

### CODE REVIEW

**Comment by jameswnl** [comment]
Is this useful in practice? I mean all our real ai service code lives in their own branches. Changes to master branch doesn't help.

**Comment by AparnaKarve** [comment]
Ideally, this should have been part of master, but since all our use case branches have already been created, we would have to propagate code with `git cherry-pick` across all other use case branches.

File: pylama.ini [hunk 1]
```diff
@@ -27,3 +27,6 @@ complexity = 10
 [pylama:pylint]
 good-names = logger,e,application,gunicorn_logger
 ignored-classes = logger
+
+[pylama:volume_type_validation/volume_type_validation.py]
+ignore = E501,C0301,R0201
```
Comment (jameswnl) [inline pylama.ini:32]: why ignoring these?
can't the offending places be fixed?

Comment (AparnaKarve) [reply pylama.ini:32]: The long line ones can be fixed, but should be done in a separate PR.

To give you some background, this was our very first use case, and mostly taken as is from the notebook implementation in gitlab.
The focus back then was to just get it integrated with the pipeline, and hence the long lines were left untouched.

Comment (AparnaKarve) [reply pylama.ini:32]: I had to ignore those for now, for pytest to run the tests in the current PR

Comment (jameswnl) [reply pylama.ini:32]: got it

**Comment by AparnaKarve** [comment]
@jameswnl I have structured the commits in such a way that only the first 3 commits would need to be applied to the other use case branches

**Review by jameswnl — APPROVED** [review]

**Pull request merged**